### ISSUE TITLE
資料夾分類

### ISSUE THREAD

**Issue by TzuChin** [hackmdio/hackmd#109]
目前雖然有標籤的功能，
但如果有資料夾分類更好

**Comment by eshao** [comment]
This would be even better if you could share the folder! That way we could have true team collaboration!

**Comment by circlehk** [comment]
It would be nice if a folder could have permission settings like a note has now. Then by setting appropriate permission to a folder and share its link to the team, others can collaborate on the same sets of notes easily. something like #322 could be also achieved easily without giving up the confidentiality of other notes

A possible workaround before the feature is available can be creating a index note which lists all the relevant notes & their URLs.

**Comment by jackycute** [comment]
@circlehk Yes, that's one of our current goals. It's not a workaround, it will be a whole new mode.

**Comment by xgdgsc** [comment]
Folder permission management would be great. Supporting single note user management would also be helpful and more flexible, something like the trello organization-board model, you can add users to a note folder with selective view/edit permissions and grant permissions to other people on some of the single notes to better collaborate temporarily outside pre-defined organization.

**Comment by furkanmustafa** [comment]
This pull request looks great!

It would also be really nice if the editor view, has a sidebar, that allows us to navigate between documents inside the folder. (Like [Confluence's Page Tree](https://confluence.atlassian.com/doc/files/139476/790795573/2/1498452974833/anatomyPage_annotated.png))

**Comment by zeigerpuppy** [comment]
It would be great if wiki style links could also be supported between pages of a collection.
This would enable using hackmd for documentation (In my opinion, with this feature hackmd would be better than [Gollum](https://github.com/gollum/gollum)).
see https://github.com/hackmdio/hackmd/issues/618
